<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SKEmbedded Video Player</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="fileInputTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1091</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="fileInputButton">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>10</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Browse..</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="optionsGroupBox">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>611</width>
      <height>681</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QGroupBox" name="colorThresholdingGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>591</width>
       <height>251</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>Thresholding Constants</string>
     </property>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>30</y>
        <width>551</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>Block Size</string>
      </property>
      <widget class="QSlider" name="blockSizeSlider">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>30</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>75</number>
       </property>
       <property name="singleStep">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>2</number>
       </property>
      </widget>
      <widget class="QLabel" name="blockSizeLabel">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>BlockSize</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>561</width>
        <height>91</height>
       </rect>
      </property>
      <property name="title">
       <string>C (subtracted from mean)</string>
      </property>
      <widget class="QSlider" name="constantThresholdSlider">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>30</y>
         <width>431</width>
         <height>41</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>50</number>
       </property>
       <property name="pageStep">
        <number>1</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>1</number>
       </property>
      </widget>
      <widget class="QLabel" name="constantLabel">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>41</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>C</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>340</y>
       <width>561</width>
       <height>23</height>
      </rect>
     </property>
     <property name="value">
      <number>24</number>
     </property>
    </widget>
    <widget class="QLabel" name="errorLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>300</y>
       <width>531</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>370</y>
       <width>521</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QGroupBox" name="comPortsGroupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>430</y>
       <width>591</width>
       <height>161</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="title">
      <string>USB Connection</string>
     </property>
     <widget class="QPushButton" name="comPortsListButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>30</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>List COM Ports</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comPortsListBox">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>381</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="comPortsConnectButton">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Connect</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>420</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>28</pointsize>
     </font>
    </property>
    <property name="text">
     <string> ▶ </string>
    </property>
   </widget>
   <widget class="QPushButton" name="goToStartButton">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>510</y>
      <width>80</width>
      <height>80</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>80</width>
      <height>80</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>22</pointsize>
     </font>
    </property>
    <property name="text">
     <string>⇐</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>69</y>
      <width>611</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>Raw Video</string>
    </property>
    <widget class="QLabel" name="rawVideoLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>581</width>
       <height>281</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>491</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>Processed 128 × 64 Video</string>
    </property>
    <widget class="QLabel" name="processedVideoLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>448</width>
       <height>224</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>448</width>
       <height>224</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>448</width>
       <height>224</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QSlider" name="playBackSlider">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>390</y>
      <width>611</width>
      <height>21</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>609</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Stream</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>636</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Invert</string>
    </property>
   </widget>
   <zorder>groupBox_3</zorder>
   <zorder>fileInputTextEdit</zorder>
   <zorder>fileInputButton</zorder>
   <zorder>optionsGroupBox</zorder>
   <zorder>playButton</zorder>
   <zorder>goToStartButton</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>playBackSlider</zorder>
   <zorder>checkBox</zorder>
   <zorder>checkBox_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
